integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeApi.kt

```diff
@@ -155,6 +158,12 @@ class CrowdNodeApiAggregator @Inject constructor(
                 }
             }
             .launchIn(configScope)
+
+        blockchainStateProvider.observeState()
+            .onEach {
+                it?.run { currentBlockHeight = bestChainHeight  }
+            }
+            .launchIn(statusScope)
```

Comment:
It might make sense to expose `currentBlockHeight` as a property (if possible) or suspend function to avoid dancing with observables. Here and in many other places, we might only need to get it once.

Reply:
Good point.

Reply:
We have a such a suspend function already.

Reply:
This has been resolved.